GooglePlayInstant/Editor/PlayInstantPublisher.cs

```diff
@@ -48,16 +48,27 @@ public static void Build()
                 return;
             }
 
```

Comment:
Move comment here:
`// Zip creation is fast enough so call jar synchronously rather than wait for post build AppDomain reset.`

Reply:
Resolved